pkg/parser/parser_readium_webpub.go

```diff
@@ -0,0 +1,84 @@
+package parser
+
+import (
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/readium/go-toolkit/pkg/asset"
+	"github.com/readium/go-toolkit/pkg/fetcher"
+	"github.com/readium/go-toolkit/pkg/manifest"
+	"github.com/readium/go-toolkit/pkg/mediatype"
+	"github.com/readium/go-toolkit/pkg/pub"
+)
+
+type WebPubParser struct {
+	client *http.Client
+	// TODO pdfFactory
+}
+
+func NewWebPubParser(client *http.Client) WebPubParser {
+	return WebPubParser{
+		client: client,
+	}
+}
+
+func (p WebPubParser) Parse(asset asset.PublicationAsset, fetcher fetcher.Fetcher) (*pub.Builder, error) {
```

Comment:
I didn't think of that convention, would definitely be helpful now that you mention it. But should it not be a comment for the `WebPubParser` struct itself since it is what implementes `PublicationParser`?

Reply:
It is on each function for two reasons:

* Pass the lint warning that requires every public function to have a doc comment (even though it would be redundant with the one declared in the interface).
* Since interface conformance is implicit, this is a way to identify which function belongs to a particular interface.